### ISSUE TITLE
It's unavailable on PHP 5.5

### ISSUE THREAD

**Issue by peter279k** [devsdmf/hello-php-extension#1]
This extension is not available on PHP 5.5,but it's worked fine on PHP 7+.

Thanks.

**Comment by devsdmf** [comment]
Hi @peter279k, 

Yes, I made this extension as an example for a [talk](https://speakerdeck.com/devsdmf/php-extending-the-core-why-you-should-learn-it-1) about PHP Extension Development that I've presented in the end of past year. I just developed this over the PHP 7.x API because its the new version of the language, and the changes in the core made the extension development API incompatible with the API of the 5.x versions.

If you have any doubts, just ask me and I'll try to answer you asap.

Best regards (:

**Comment by peter279k** [comment]
I think we sould add some build tips in ```README.md```.

**Comment by devsdmf** [comment]
Yes, would be a good thing, but for now, I'm working on some other projects, so feel free to submit a PR! (:
